### PR TITLE
config class + automatic update check

### PR DESCRIPTION
**Pull request by Chive**

### CODE REVIEW

File: aldryn_client/cli.py [hunk 1]
```diff
@@ -362,30 +366,27 @@ def version(skip_check, show_error):
...
+                "now using 'pip install aldryn-client=={newest_version}'"
+                .format(newest_version=newest_version)
+            )
+        elif newest_version is False:
             if show_error:
                 click.secho(
-                    '\nThere was an error while trying to retrieve the latest '
-                    'version from pypi.python.org:\n', fg='red'
+                    'There was an error while trying to retrieve the latest '
+                    'version from pypi.python.org',
```
Comment (stefanfoulis) [inline aldryn_client/cli.py]: The wording could be misunderstood as it actually trying to download the package.

**Comment by stefanfoulis** [comment]
nice :-) :+1: 
:hammer:

File: aldryn_client/cli.py (hunk 1 above)
Comment (Chive) [inline aldryn_client/cli.py]: how about
```python
                click.secho(
                    'There was an error while trying to check for the latest '
                    'version on pypi.python.org',
                    fg='red'
                )
```

Comment (stefanfoulis) [inline aldryn_client/cli.py]: ``'There was an error while trying to check for the latest version on pypi.python.org'`` :+1:

**Pull request merged**